src/librehatti/suspense/forms.py

```diff
@@ -154,12 +154,11 @@ class Meta:
         widgets = {'session': forms.HiddenInput(),\
         'voucher_no': forms.HiddenInput()}
 
-    choices=[ (o.id, str(o)) for o in FinancialSession.objects.all()]
-    receipt_session = forms.ChoiceField(choices)
-
```

Comment:
The error was "django.db.utils.ProgrammingError: (1146," Table 'librehatti.voucher_financialsession' doesn't exist") "
I think whenever we try to migrate database,  django tries to find financial session table for a form field which django wasn't able to find.